python-data-analysis/python-data-analysis-core/analyzing-ii/tvshow-ratings.md

```diff
@@ -125,8 +125,8 @@ people\
 - Age
 - .sort_values
 - ascending=False
-- to_frame
-- reset_index
+- .to_frame
+- .reset_index
 - index
 - to_data_frame
```

Comment:
all the correct answers start with a `.` so this one should too
```suggestion
- .to_data_frame
```

Reply:
Actually, that is a wrong answer, the correct method is to_frame, not to_data_frame, which is above this one

Reply:
Should I still add the dot?

Reply:
I think the to_data_frame is related to the R language. It was added as a wrong answer

Reply:
> Actually, that is a wrong answer, the correct method is to_frame, not to_data_frame, which is above this one

yes, that was my point

if all the correct answers start with a dot it would be very easy to identify the _wrong_ method by looking at the ones without a dot

if you saw this question:

```
what is the correct method?

obj???

- .correctMethod()
- wrongMethod1()
- wrongMethod2()
```

you'd notice the correct answer without thinking about the question

Reply:
Yeah, that makes a lot of sense, my brain just went in a different direction when editing this 😅